schema/url/pass/plugin-histogram.raml

```diff
@@ -13,8 +13,7 @@ types:
       outputEntityId:
         type: string
       showMissingness:
-        type: string
-        enum: ['TRUE', 'FALSE']
+        type: ShowMissingnessNoAxes
```

Comment:
this one i expected to see ShowMissingness. Sorry i'm not following 😄 What's the criteria for ShowMissingness vs ShowMissingnessNoAxes?